src/validation/validate.ts

```diff
@@ -95,6 +96,16 @@ export function validate(options: BeachballOptions, validateOptionsOverride?: Pa
       );
       process.exit(1);
     }
+    if (
+      !change.dependentChangeType ||
+      !isValidDependentChangeType(change.dependentChangeType) ||
+      disallowedChangeTypes?.includes(change.dependentChangeType)
```

Comment:
disallowedChangeTypes check should be passed in the "isValidDependencyChangeType" so it can do the filtering there